nixos/modules/services/networking/corerad.nix

```diff
@@ -81,6 +81,7 @@ in {
         NotifyAccess = "main";
         ExecStart = "${getBin cfg.package}/bin/corerad -c=${cfg.configFile}";
         Restart = "on-failure";
+        RestartKillSignal = "SIGHUP";
```

Comment:
this should probably be a different commit:

```
corerad: 0.2.7 -> 0.2.8
nixos/corerad: set RestartKillSignal
```
not sure what the subject for the module should be.

Reply:
Not a problem, done!